Aztec/Classes/Private/Libxml2/Data/ElementNode.swift

```diff
@@ -93,9 +20,9 @@ extension Libxml2 {
         private(set) var attributes = [Attribute]()
         private(set) var children: [Node]
 
-        private var standardName: StandardName? {
+        private var standardName: HTMLElementType? {
```

Comment:
We've lost clarity here given the name has no relation to the type.  It'd be much cleaner here if we had:

```
private var standardType: StandardElementType? {
```

The word "standard" would give a hint as to why this property can be `nil` (ie: if the type is not standard, it will be `nil` here).